chaos-monkey-spring-boot/pom.xml

```diff
@@ -120,6 +120,14 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>no.finn.unleash</groupId>
+            <artifactId>unleash-client-java</artifactId>
+            <version>4.2.1</version>
+            <optional>true</optional>
+        </dependency>
```

Comment:
OK. I think I get it now. We didn't put the spring-security dependencies in the `chaos-monkey-depedencies`, because Chaos Monkey isn't dependent on it, but Unleash is.